pkg/logstructured/logstructured.go

```diff
@@ -302,7 +304,8 @@ func (l *LogStructured) ttl(ctx context.Context) {
 				return
 			case <-time.After(time.Duration(event.KV.Lease) * time.Second):
 			}
-			l.Delete(ctx, event.KV.Key, event.KV.ModRevision)
+			_,_,_,err := l.Delete(ctx, event.KV.Key, event.KV.ModRevision)
```

Comment:
In an HA setup this might cause a bunch of spurious errors.  You'd have to double check that a delete of a something that doesn't exist will thrown an error.

Reply:
What would be a good way to check that?